test/test.jl

```diff
@@ -1,16 +1,28 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-using Base.Test
```

Comment:
I don't care for it – it's boilerplate. I would be ok with importing `Base.Test` by default, however.